app/src/main/java/com/pajato/android/gamechat/common/BaseFragment.java

```diff
@@ -283,8 +283,6 @@ protected boolean updateAdapterList() {
         switch (type) {
```

Comment:
Since we have crossed the magic number threshold (6-7) for switch cases, I am now thinking that we should this into the FragmentType class as a FragmentType#getListItemData() method, or perhaps even better, implement a getListItemData(signature) method in each fragment.  Your thoughts?